ext/standard/php_random.h

```diff
@@ -19,6 +19,10 @@
 #ifndef PHP_RANDOM_H
 #define PHP_RANDOM_H
 
+#include <Zend/zend_portability.h>
```

Comment:
By convention, this should be `#include "zend_portability.h"`. `<>` is typically only used for system headers.